src/actions/schedule-actions.js

```diff
@@ -80,4 +127,12 @@ export const getShareLink = (filters, view) => {
 
 export const callAction = (key, action, payload) => (dispatch) => {
     return dispatch(createAction(action)({...payload, key}));
+};
+
+export const reloadScheduleData = () => (dispatch, getState) => {
+    const {userState, loggedUserState} = getState();
+    const {isLoggedUser} = loggedUserState;
+    const {userProfile} = userState;
+    dispatch(createAction(RELOAD_SCHED_DATA)({isLoggedUser, userProfile }));
```

Comment:
Actually, is isLoggedUser payload used at all?

Reply:
yes i do think so